tester/include/game_tester/random_mock.hpp

```diff
@@ -0,0 +1,49 @@
+#include <openssl/rand.h>
+
+namespace testing::random_mock
+{
+
+constexpr static uint32_t rand_initializer = 32;
+
+static int stdlib_rand_seed(const void *buf, int num) {
+    assert(num >= sizeof(unsigned int));
+    std::srand( rand_initializer );
```

Comment:
It's no need to call srand here, because we set seed globally in main(): https://github.com/DaoCasino/game-contract-sdk/blob/3c9128470d381d7a65f8db7f0c5656a6ed322c27/tester/include/game_tester/game_tester.hpp#L396